.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: minimal
+
+services: docker
+
+install:
+    - make build
```

Comment:
```suggestion
    - travis_retry make build
```

Reply:
I'm not sure about this. I would want to know if there are problems during build.

Reply:
It retries 3 times. Useful if there's temporary network timeouts.

Reply:
Ok. Will update.